Gesture_control.py

```diff
@@ -43,6 +43,21 @@ def find_contours(thresh):
     contours,heirarchy = cv2.findContours(thresh,cv2.RETR_TREE,cv2.CHAIN_APPROX_SIMPLE) #give list of all essential boundary points
     return contours
 
+def centroid(contours):
```

Comment:
Pass a single contour whose centroid is to be calculated to the centroid function.
You can create another function to find the max_contour